R/load.R

```diff
@@ -29,11 +29,7 @@ load <- function(project = NULL, quiet = FALSE) {
 
   renv_scope_error_handler()
 
-  project <- normalizePath(
-    project %||% renv_project_find(project),
-    winslash = "/",
-    mustWork = TRUE
-  )
+  project <- renv_project_resolve(project)
```

Comment:
Do we want to preserve the `renv_project_find()` behavior here? (I think we do)

Reply:
Yeah, I'll revert.